go/client/ctclient/ctclient.go

```diff
@@ -70,12 +75,33 @@ func addChain(logClient *client.LogClient) {
 	fmt.Printf("%v\n", signatureToString(&sct.Signature))
 }
 
+func getRoots(logClient *client.LogClient) {
+	roots, err := logClient.GetAcceptedRoots(context.Background())
```

Comment:
Done; I've added context to the other functions too, although full plumbing will await issue #1326.